data/feature/en/text.yaml

```diff
@@ -62,6 +62,6 @@
 - title: More than (just) alias
   desc: >
     SimpleLogin's ambition is to protect your online privacy and email is only the beginning. <br>
-    Stay tuned for upcoming features like single sign-on, phone number protection and credit card protection.
+    Stay tuned for upcoming features like phone number protection and credit card protection.
```

Comment:
We only need to update the english version, other ones will be updated later by localization team.